sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/DigitalTwinsAddRelationshipOptions.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.DigitalTwins.Core
+{
+    /// <inheritdoc />
+    [CodeGenModel("DigitalTwinsAddRelationshipOptions")]
+    public partial class DigitalTwinsAddRelationshipOptions
+    {
+        // This class declaration changes the namespace; do not remove.
```

Comment:
Would you check with Pavel if there is a way in autorest.md to change the namespace so we don't have to do this nonsense?

Reply:
ah, we already are. I don't think we need these.

Reply:
@pakrym is there any short cut for this? We are declaring these classes in our customized folder so that we can change the namespace. Is there an autorest config for .NET that works like how Java has:
```yaml
use: '@autorest/java@4.0.3'
java:
    custom-types-subpackage: models
    custom-types: DigitalTwinModelsAddOptions
```

Which automatically puts DigitalTwinModelsAddOptions in the models subpackage

Reply:
You are in luck indeed https://github.com/Azure/autorest.csharp#exclude-models-from-namespace

If that was what you are asking for.

Reply:
Unfortunately we only want to change the namespace of most of the models, not all of them. I don't see a better way other than individually declaring these custom options classes like I already do. I'll leave this as is for now